### PR TITLE
PR for homework from Kyle

### PR DESCRIPTION
**Pull request by iamkhaihoang**

### CODE REVIEW

**Comment by iamkhaihoang** [comment]
Commit merged homework from Kyle

File: variables.tf [hunk 1]
```diff
@@ -128,4 +128,53 @@ variable "s3_tag" {
...
+}
+
+variable "db_password" {
+  default = "ziyotek12345"
+}
+
+variable "db_skip_final_snapshot" {
+  default = true
+}
+
```
Comment (rpaskalev) [inline variables.tf:165]: that wont work here, only hard coded values are accepted. arguments cannot be referenced in vars

File: variables.tf [hunk 2]
```diff
@@ -128,4 +128,53 @@ variable "s3_tag" {
...
+variable "db_subnet_group_name" {
+  default = aws_db_subnet_group.ziyotek_db_sg.id
+}
+
```
Comment (rpaskalev) [inline variables.tf:169]: same here

File: variables.tf [hunk 3]
```diff
@@ -128,4 +128,53 @@ variable "s3_tag" {
...
+variable "db_vpc_security_group_ids" {
+  default = [aws_security_group.ziyotek_devops_all_all.id]
+}
+
+variable "db_sg_name" {
+  default = "ziyotek"
+}
+
```
Comment (rpaskalev) [inline variables.tf:177]: same here

**Review by rpaskalev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**